app/menu.js

```diff
@@ -39,22 +59,22 @@ export default class MenuBuilder {
       submenu: [
         {
           label: 'About Bob',
-          selector: 'orderFrontStandardAboutPanel:'
+          role: 'about'
         },
         {type: 'separator'},
         {label: 'Services', submenu: []},
         {type: 'separator'},
         {
           label: 'Hide Bob',
           accelerator: 'Command+H',
-          selector: 'hide:'
+          role: 'hide'
```

Comment:
Are the colons you removed here causing issues? Curious what these changes fix.

Reply:
We can remove this commit if you like. What happened was, I was reviewing the existing menu code and couldn't find documentation for the `selector` property. Then I found this: https://github.com/electron/electron/pull/3554 indicating that `selector` was deprecated and replaced by `role`. The available values for `role` are documented here: https://www.electronjs.org/docs/all#roles

Reply:
Nope, works for me. Glad this has been updated.